### PR TITLE
Add method for updating envelope references on a case

### PR DESCRIPTION
**Pull request by rkondratowicz**
https://tools.hmcts.net/jira/browse/BPS-1373

### CODE REVIEW

**Review by aliveni — APPROVED** [review]
It would be nice to rename **updateUpdate**EnvelopeReferences() method name but I can't think of any better name

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/CaseDataUpdater.java [hunk 1]
```diff
@@ -58,6 +70,26 @@
         return updatedCaseData;
     }
 
+    @SuppressWarnings("unchecked")
+    public Map<String, Object> updateUpdateEnvelopeReferences(
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/CaseDataUpdater.java:74]: updateEnvelopeReferencesForCaseUpdate ?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/CaseDataUpdater.java:74]: done

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/CaseDataUpdater.java:74]: If we have CaseAction then can we just say updateEnvelopeReferences ?

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**